scalableminds/webknossos
# Set default view configuration in datasource properties json

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
for the evolutions:
- after the evolution the segmentationOpacity of the configuration element should be gone
- instead the old segmentationOpacity value should be the alpha value of a newly created entry in the layers dict with the name of the related segmentation layer as key
- the reverse evolution should take the alpha value from segmentation layer entry in the layers dict and save it as segmentationOpacity
- to confirm that the configuration entries are still correct, open the dataset edit view and look at the view configurations

for the disk defaults:
- if a view configuration is supplied in the datasource properties json, it should take precedence over the normal default, but if a existing dataset config takes precedence over the disk default
- e.g. change the `alpha` value of a color layer (default is 100) 


### Issues:
- fixes #4298 
- fixes #4378 

------
